steps/Asset.steps.ts

```diff
@@ -46,11 +54,13 @@ Then('Techsupport Employee Creates Item',async () => {
  await asset.item(testConfig.Item)
  await asset.code(testConfig.Itemcode)
  await asset.addbutton()
- await utils.assertSucessMsg(testConfig.updatedtoastmessage)
  await asset.close()
+ 
  })
```

Comment:
Before creating MR, please fix the format

Reply:
Format fixed